src/xunit.performance.execution/xunit.performance.execution.csproj

```diff
@@ -7,7 +7,7 @@
   <PropertyGroup>
     <AssemblyTitle>xunit.performance.execution</AssemblyTitle>
     <Description>xUnit Performance Execution</Description>
-    <TargetFramework>netstandard1.3</TargetFramework>
+    <TargetFramework>netstandard1.5</TargetFramework>
```

Comment:
It has been the case from the start. People using the API, use the API package directly, which is 1.5